packages/web/src/components/CheckboxField/_CheckboxField.scss

```diff
@@ -0,0 +1,90 @@
+@use 'theme';
+@use '../../tools/typography';
+
+.CheckboxField {
+    max-width: 100%;
+}
+
+.CheckboxField__label {
```

Comment:
Long labels should not wrap around the input:

<img width="370" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/156638391-5c5cb946-c647-4514-99cf-4f6226009394.png">

<img width="335" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/156638478-52822f9d-3dcc-4a7e-b860-2841a48d66f2.png">

Reply:
Fixed.